src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { useState, useEffect, memo } from 'react';
+
+import { useParams } from 'react-router-dom';
+import { Loader } from '../Loader/Loader';
+import { PeopleTable } from '../PeopleTable/PeopleTable';
+import { getPeople } from '../../api';
+import { Person } from '../../types/Person';
+
+export const PeoplePage: React.FC = memo(() => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [isError, setIsError] = useState(false);
+  const [isLoadingFinish, setIsLoadingFinish] = useState(false);
+  const { selectedUser = '' } = useParams();
```

Comment:
you should get selectedUserSlug inside component with person details, this parameter is redundant here.